### PR TITLE
Add wrapper for ConvTranspose2d

### PR DESCRIPTION
**Pull request by lhw362950217**

### CODE REVIEW

File: cgotorch/cgotorch.cc [hunk 1]
```diff
@@ -37,6 +37,19 @@ char *RandN(int rows, int cols, int require_grad, Tensor *result) {
   }
 }
 
+char *RandNByShape(int64_t *shape, int64_t shape_len, int require_grad,
```
Comment (QiJune) [inline cgotorch/cgotorch.cc:40]: `RandN` is supported by shape in #74 . We could remove `RandNByShape` after #74 is merged

File: tensor_test.go [hunk 2]
```diff
@@ -10,3 +10,17 @@ func ExampleTensor() {
 	t.Close()
 	// Output:
 }
+
+func ExampleTranspose2d() {
```
Comment (Yancey1989) [inline tensor_test.go:14]: Please rename the function name should begin with **Test**  word, please rename to `TestExampleTranspoose2d(t  
 *testing.T)` .

File: tensor_test.go [hunk 3]
```diff
@@ -10,3 +10,17 @@ func ExampleTensor() {
...
+	weight := torch.RandN([]int{1, 3, 3}, false)
+	var bias torch.Tensor
+	stride := []int{1}
+	padding := []int{0}
+	outputPadding := []int{0}
+	groups := 1
+	dilation := []int{1}
+	out := torch.ConvTranspose2d(input, weight, bias,
+		stride, padding, outputPadding, groups, dilation)
+	out.Print()
```
Comment (Yancey1989) [inline tensor_test.go:25]: printing the Tensor has no help to check the function work well, do we have another way to check it?

Comment (lhw362950217) [reply tensor_test.go:25]: As we can't specify values to tensor now, we can't validate the output. So maybe, we can only test if there is no error occurs when running test.

**Pull request merged**